Pertempto/Lists
# feat: improve editing experience

## What has changed

Replaced the EditItemDialog with a toolbar on the bottom of the list page.

## Connected Issues


Closes #43

## QA Steps

1. Open a list
2. Try to add/edit/delete items.
    - You should be able to do ALL the things you could do before with the edit item dialog.
